From 53e3e37e91adf6b38fe4057a93131a21c04052b2 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:38 -0400
Subject: [redhat] kabi: Adding symbol blk_fetch_request (block/blk-core.c)

Message-id: <5744072aa12dd3166b1e4d436ef173976a1d399d.1410510773.git.poros@redhat.com>
Patchwork-id: 96052
O-Subject: [PATCH RHEL7.1 038/233] kabi: Adding symbol blk_fetch_request (block/blk-core.c)
Bugzilla: 1062361 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Fetch a request from a request queue.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_fetch_request
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_fetch_request
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_fetch_request

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_fetch_request b/redhat/kabi/kabi-module/kabi_ppc64/blk_fetch_request
new file mode 100644
index 0000000..f3ff14a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_fetch_request
@@ -0,0 +1,2 @@
+#1-
+0xd7d260c1 blk_fetch_request vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_fetch_request b/redhat/kabi/kabi-module/kabi_s390x/blk_fetch_request
new file mode 100644
index 0000000..4df6184
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_fetch_request
@@ -0,0 +1,2 @@
+#1-
+0xe4d031a9 blk_fetch_request vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_fetch_request b/redhat/kabi/kabi-module/kabi_x86_64/blk_fetch_request
new file mode 100644
index 0000000..e82d100
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_fetch_request
@@ -0,0 +1,2 @@
+#1-
+0x605420f9 blk_fetch_request vmlinux EXPORT_SYMBOL
-- 
1.7.1